bokehjs/package-lock.json

```diff
@@ -1136,7 +1136,6 @@
       "requires": {
         "anymatch": "1.3.2",
         "async-each": "1.0.1",
-        "fsevents": "1.1.3",
```

Comment:
@marcuskd, please revert changes to this file. You will also need to update npm to version >= 5.6 to avoid such issues (anything below results in undeterministic module resolution). This will be enforced on build-level after PR #7575.

Reply:
Hi @mattpap. Changes reverted and pushed a moment ago. I didn't notice I'd done this, will update npm as suggested.